_layouts/english.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+---
+
+<div class="cover-photo">
+  <img src="/assets/img/2019_all.png" alt="Mitou Junior Photo in 2019" class="top-img" style="margin-bottom: -8px;">
+
+  <div class="page-head">
+    <h1>{{ page.title }}</h1>
+  </div>
+</div>
+
+{{ content }}
```

Comment:
To make English page I18n we need to move `{{ content }}` out of `<div class="post">` section so that we can apply same CSS for both English and Japanese pages in just one line (`{% include sponsors.html %}`). 🎨👨‍🎨✨